### PR TITLE
refactor(notification): use aurelia-pal to set default container

### PR DESCRIPTION
**Pull request by doktordirk**

### CODE REVIEW

File: src/notification.js [hunk 1]
```diff
@@ -34,9 +35,9 @@ export class Notification {
     this.setContainer();
     let aureliaComposedListener = () => {
       this.setContainer();
-      document.removeEventListener('aurelia-composed', aureliaComposedListener);
+      DOM.removeEventListener('aurelia-composed', aureliaComposedListener);
```
Comment (RWOverdijk) [inline src/notification.js:38]: No DI?

Comment (doktordirk) [inline src/notification.js:38]: DOM is DOM. DOMs an abstraction for document for browser compatibility. basically never use window or document, but DOM and PLATFORM(?or so)

Comment (doktordirk) [inline src/notification.js:38]: so behind the scences the code is unchanged. just (my) best practise

Comment (RWOverdijk) [inline src/notification.js:38]: Ah yeah, just that DI allows you to even overwrite DOM with a different implementation. That's why DI is cool. But I do understand your reasoning, so I'll :+1:

**Comment by RWOverdijk** [comment]
:+1:

File: src/notification.js (hunk 1 above)
Comment (doktordirk) [inline src/notification.js:38]: could be tricky since aurelia attached the aurelia-composed event to pals DOM

**Pull request merged**